src/DIRAC/DataManagementSystem/Client/FileCatalogClientCLI.py

```diff
@@ -2245,8 +2245,7 @@ def do_rebuild(self, args):
            rebuild <option>
         """
 
-        argss = args.split()
-        _option = argss[0]
+        _argss = args.split()
```

Comment:
just replace `args` with `_args` in the function siganture and drop this split"?